extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -320,6 +320,11 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 			}
 		}
 
+		// Delete existing Python variables in ADS to prevent conflict with other installs
+		delete process.env['PYTHONPATH'];
```

Comment:
Might as well move these deletes to the top of the method. That would prevent any command ordering edge cases if we modify this method later on.